test/Test/BaseDAO/Limits.hs

```diff
@@ -0,0 +1,66 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+{-# LANGUAGE PartialTypeSignatures #-}
+{-# OPTIONS_GHC -Wno-partial-type-signatures #-}
+
+-- | Checks on that our contract fits into reasonable limits.
+module Test.BaseDAO.Limits
+  ( test_Operation_size
+  ) where
+
+import Universum
+
+import Fmt (pretty)
+import Test.HUnit (Assertion, assertFailure)
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase)
+
+import Lorentz.OpSize
+import Lorentz.Value
+import Tezos.Address
+
+import qualified Lorentz.Contracts.BaseDAO as DAO
+
+originationOpSize :: _ => DAO.Config pm -> DAO.Storage pm -> OpSize
```

Comment:
Why do we need `_ =>`?

Reply:
That constraint is extremely big and for the module-local method we don't care.